### DISCUSSION TITLE
Is the project dead?

### DISCUSSION THREAD

**Discussion by nlogozzo** [Kinnara/ModernWpf#369] [General]
Is this project dead? We haven't seen a commit in 3 months...which is ok, but there are a ton of unanswered issues and bugs. What happened to this project? It's used by almost every WPF project I know now a days.

**Comment by ghost1372** [comment]
I think is dead, mybe because of Project reunion

**Reply by nlogozzo** [reply]
They said that wouldn't happen tho as of #181 
@ShankarBUS Do you know anything?

**Reply by ShankarBUS** [reply]
Unfortunately, this project seems to be no longer maintained.

I am just a consumer of this project just like everyone else. My app depends on this package. I can't give my opinions.

Maybe @Kinnara could answer your question.

**Comment by ShankarBUS** [comment]
I am not sure whether the author will add new features or not (i.e. WinUI 2.6 styles and new controls).

You can always switch to WinUI 3 or WinAppSDK (aka Project Reunion). Sticking with ModernWpf is your choice. But WPF itself is put to maintenance mode.

So, I don't think it is a viable option anymore. Just my opinions.

**Reply by nlogozzo** [reply]
I mean hasn't WPF been on maintenance mode for ever lol? We really haven't seen new features in a while

**Reply by ShankarBUS** [reply]
😂😂😂

**Comment by nlogozzo** [comment]
If your up for it @ShankarBUS maybe me and you can fork this repo and keep it up to date, update the styles, add new controls, fix the bugs, etc...
And whoever else may be open to it.

**Reply by ShankarBUS** [reply]
I would love to do that. But I can't even maintain my app at this stage. Someone else could be willing to do.

TBH, I would recommend moving to technologies that are being actively developed and maintained properly by some great contributors. Such as WinUI.

This project is mainly being maintained by a single person i.e. the author @Kinnara. And they single handedly maintained it for more than a year and a half now.

They might have personal or work issues which renders them unable to focus on this project. Which I can completely relate to and understand.

Only if this project had enough regular contributors, it might have survived longer.

**Reply by ghost1372** [reply]
Currently the only reason to use this library is to support Windows 7 I have recently decided to move my apps to Réunion. But it would be great if this project could continue, I noticed that PowerToys also uses this library for the user interface

**Reply by nlogozzo** [reply]
Yeah I already have apps with the WindowsAppSDK, but it just feels too clunky to me as compared to solid WPF. But I guess it's the future and Microsoft will eventually work out the kinks...hopefully 🤷‍♂️

**Reply by alkalinin** [reply]
I am developing 3D application with WPF. Currently WindowsAppSDK does not support 3D graphics like WPF. So my choice is the WPF. And also WPF github repository is still alive and quite active. 

These WPF controls I use a lot and I think we should help to keep this repository alive.

**Comment by alxkalinin** [comment]
Just for information. I suppose that now the community efforts for updating WPF UI is concentrated over https://github.com/lepoco/wpfui repository.

**Comment by wherewhere** [comment]
It's a pity that this project has stopped updating. I am trying to update it to Win11 Style. I was developing UWP before, and I'm not too familiar with WPF. So the update will be very slow. If you are interested you can also come along and update [it](https://github.com/wherewhere/ModernWpf). I have updated Button and Expander yet.
![mmexport1648559649931.png](https://user-images.githubusercontent.com/27689196/160619666-9df10401-0dea-4408-8e92-57c2f0591f81.png)

**Reply by wherewhere** [reply]
It has a new calendar now!
![mmexport1649246772452.png](https://user-images.githubusercontent.com/27689196/161981778-584e3f0d-ed0f-491d-8e89-9196eb9c844f.png)
![mmexport1649246777315.png](https://user-images.githubusercontent.com/27689196/161981789-dd976ed0-f1a0-453b-a14a-109e6b3b538d.png)

**Reply by wherewhere** [reply]
#446
